doc/readme/EXAMPLES.md

```diff
@@ -35,8 +35,14 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example. Just create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
+It is extremely simple to add a new example: simply create another `*.sg` file.
+We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).  
+
+The following are the steps required to port all existing examples to meta examples:  
+
+1. Just select any of the existing python examples ```examples/uncodcumented/python_modular```   
```

Comment:
`... existing python examples under`

Reply:
btw a simple \`this\` is enough it's not a listing block..

Reply:
@vigsterkr Really thank you for suggesting changes. I have updated the PR.

Reply:
@vigsterkr Please let me know if any further updations are required.